specs/commandline/plugin.md

```diff
@@ -36,15 +36,55 @@ Aliases:
   list, ls
 ```
 
+### notation plugin install
+
+```text
+Installs a plugin
+
+Usage:
+  notation plugin install --name <plugin name> --plugin-package <plugin package> [flags]
+
+Flags:
+  -n, --name           string   name of the plugin
+  -p, --plugin-package string   path to the plugin package
+  -h, --help                    help for install
+
+Aliases:
+  install, add
+```
+
+### notation plugin remove 
+
+```text
+Removes a plugin
+
+Usage:
+  notation plugin remove <plugin name> [flags]
+
+Flags:
+  -h, --help          help for remove
+
+Aliases:
+  remove, rm
```

Comment:
How about adding `uninstall` to the aliases?

Reply:
I'm not sure how many aliases are too many, but I added uninstall and delete. :)